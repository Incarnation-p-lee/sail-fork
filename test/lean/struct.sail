default Order dec

$include <prelude.sail>

struct My_struct = {
  field1 : int,
  field2 : int,
}

