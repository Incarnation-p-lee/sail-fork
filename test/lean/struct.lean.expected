structure My_struct where
  field1 : Int
  field2 : Int
